docs/new_project_guide.md

```diff
@@ -79,6 +91,11 @@ reproducing and fixing bugs than standard one outlined in the reproducing guide.
 
 Example: [boringssl](https://github.com/google/oss-fuzz/blob/master/projects/boringssl/project.yaml).
 
+### experimental
+A boolean (either True or False) that indicates whether this project is in evaluation mode. This allows a project to be
+fuzzed and generate crash findings, but not file them in the issue tracker. The crashes can be accessed on [ClusterFuzz homepage](clusterfuzz.md#web-interface). This should be only used if you are not a maintainer of the project and have
+less confidence in the efficacy of your fuzz targets.
```

Comment:
Not sure if the last sentence is needed. Why a maintainer can't try e.g. MSan in an experimental mode?

Reply:
This experimental flag is for root, meaning if added, can disabled bug filing for entire project. https://chrome-internal-review.googlesource.com/c/chrome/tools/clusterfuzz/+/655014